jsapp/js/components/list.es6

```diff
@@ -154,55 +154,69 @@ export class ListCollectionFilter extends React.Component {
       availableCollections: [],
       collectionsLoaded: false,
     };
+    this.unlisteners = [];
     autoBind(this);
   }
-  componentDidMount () {
+
+  componentDidMount() {
+    this.unlisteners.push(
+      actions.library.getCollections.completed.listen(this.onGetCollectionsCompleted)
+    );
     this.queryCollections();
   }
-  queryCollections () {
-    dataInterface.getCollections().then((collections)=>{
-      var availableCollections = collections.results.filter((value) => {
-        return (
-          value.access_types &&
-          !value.access_types.includes(ACCESS_TYPES.public)
```

Comment:
Were we trying to exclude `public` access types before but not now? I'm guessing it's because public collections is now live but just want to be sure

Reply:
Yeah that's the reason :)